development/lib/run-command.js

```diff
@@ -1,4 +1,5 @@
 const spawn = require('cross-spawn');
+const shellwords = require('shellwords');
```

Comment:
I think this is an unrelated change?

Reply:
Yeah, I just was trying to debug CI failures better, but will split this off into its own PR.